tests/unit/test_fast_check.py

```diff
@@ -24,7 +24,7 @@ def test_token_masking_old_format():
 def test_token_masking_new_format():
     assert_that(obfuscate_dynatrace_access_key(
         "dt0c01.FAKE11.FAKEEU7QQ5KW7BZVN2XQFAKEFAKEJFHLZKTOUTFZKAAZTRSFR4HTJXZKXIZPPUVZ2D7YPRXY2IUFAKE7JY")).is_equal_to(
-        "dt0c01.FAK***********************************************************************************7JY")
+        "FAKE11")
```

Comment:
I'd expect to log 'dt0c01.FAKE11' - that's what we do elsewhere (cluster UI, debugUI, restAPI) & it does make you easily recognize this is the token + that it us the new format token.

Reply:
Ok, will do it